identity/Cargo.toml

```diff
@@ -50,6 +50,9 @@ send-sync-storage = ["identity-account-storage/send-sync-storage"]
 # Exposes Storage `test_suite` module.
 storage-test-suite = ["identity-account-storage/storage-test-suite"]
 
+# Enables encryption and decryption functionality
+encryption-decryption = ["identity-account/encryption-decryption"]
```

Comment:
Change the name here as well, if you decide to change it for the `account` crate. Maybe it makes sense to use `account-encryption` for clarity? Not sure, I leave the decision to you.